tkweb/apps/drinks/templates/drinks/download.html

```diff
@@ -0,0 +1,8 @@
+
+{% extends "drinks/base.html" %}
+{% block fulltitle %}Drinks{% endblock %}
```

Comment:
Kun forsiden af appen bør bruge `fulltitle`. Her bør du i stedet skrive:
```suggestion
{% block title %}Download{% endblock %}
```